### ISSUE TITLE
Can't killall firejailed chromium

### ISSUE THREAD

**Issue by Harvie** [netblue30/firejail#1385]
I've executed chromium like `firejail chromium` and now i can't kill it with `killall -9 chromium` even as root, says `chromium: no process found`. Without firejail it works just ok. While `pkill -9 chromium` seems to work.

**Comment by netblue30** [comment]
In a terminal start chromium: "firejail chromium".

In a different terminal, do a "firejail --tree". What does it say? If "firejail --tree" finds it, killall should also find it.

**Comment by Harvie** [comment]
The output is rather long, but it correctly litsts all chromium processes from any terminal i've tried. Also ps aux shows it. But it still can't be killed using killall.

**Comment by netblue30** [comment]
tested on Debian stable, it seems to be working. It could be a problem with the newer killall you have on Arch. Run a "killall --version", over here I have 22.21

**Comment by Harvie** [comment]
```
killall (PSmisc) 23.1
Copyright (C) 1993-2017 Werner Almesberger and Craig Small

PSmisc comes with ABSOLUTELY NO WARRANTY.
This is free software, and you are welcome to redistribute it under
the terms of the GNU General Public License.
For more information about these matters, see the files named COPYING.
```

maybe it's some protection that prevents killall from killing processes in LXC containers when managing host machine.

**Comment by Harvie** [comment]
Ha! It works when using `killall --ns 0`.

```
       -n, --ns
              Match against the PID namespace of the given PID. Use 0 to match against all namespaces. The default is to  match  against
              the current PID namespace.
```

**Comment by netblue30** [comment]
So, basically they added namespace support to killall. I'll add it to FAQ, thanks.

**Comment by chiraag-nataraj** [comment]
@netblue30 Where do you want to put this info?

**Comment by matu3ba** [comment]
@netblue30 Reminder to put #1385 into [FAQ](https://firejail.wordpress.com/support/).

**Comment by chiraag-nataraj** [comment]
@matu3ba Or, we can put it in the Wiki! Actually, we could transition the FAQ to the Wiki, if that's okay with @netblue30? That way, the maintenance burden isn't just on one person :)

**Comment by matu3ba** [comment]
@chiraag-nataraj Sure. Could you ask netblue30 to give me Collaborator Access?
Closing and (nicely) tagging issues would be much easier.

**Comment by chiraag-nataraj** [comment]
@matu3ba I believe the wiki is open for all to edit? But please follow the directions on the home page of the wiki for adding new pages and we can take it from there :slightly_smiling_face:

**Comment by matu3ba** [comment]
Closed and refined in #2792
